service/http/types.go

```diff
@@ -18,6 +18,25 @@ type ReqCreateDistribution struct {
 	PackTemplate ReqPackTemplate    `json:"packTemplate"`
 }
 
+type ReqCreatePack struct {
+	/*
+		{
+			distId: // long lived distribution
+			nft_flow_ids: []string // nft_flow_id
+			hash: // commitHash
+			issuer // ie: NBA PackNFT address
+			packReference: {
+				"name": "PackNFT",
+				"address": "4dfd62c88d1b6462",
+			}
+		}
+	*/
+	DistributionID uuid.UUID         `json:"distID"`
+	NFTFlowIDs     common.FlowIDList `json:"nftFlowIds"`
+	CommitmentHash string            `json:"commitmentHash"`
+	PackReference  AddressLocation   `json:"packReference"`
+}
```

Comment:
Do we need an additional param for the address we're minting the pack to?

Reply:
Hmm good question, right now it's the distribution's issuer:
https://github.com/dapperlabs/flow-pds/blob/53c7ea246773061d6feebe36af8fb32f3077d665/service/app/contract_service.go#L455

I think it makes sense for the issuer to have ownership of the pack for now 🤔 
It could also be the user that's requesting the pack to be minted

Reply:
Sounds good to me 
If we're converting on the fly, does it make sense to have the recipient be the user? 
Or we mint into the issuer account because it's going to be listed anyway 

If it's not a big lift, making that a param might make it more flexible for us later

Reply:
in the case where this endpoint is called on behalf of a user creating a pack, would a distributionID need to be specified?  and would a user's account have the appropriate collection/capability to receive the pack?